service/src/test/java/com/epam/rft/atsy/service/impl/ApplicationsServiceImplTest.java

```diff
@@ -82,6 +86,44 @@
   @InjectMocks
   private ApplicationsServiceImpl applicationsService;
 
+  @Test(expected = IllegalArgumentException.class)
+  public void getApplicationShouldThrowIllegalArgumentExceptionWhenIdIsNull() {
+    //Given
+
+    //When
+    ApplicationDTO result = applicationsService.getApplication(null);
+
+    //Then
+  }
+
+  @Test
+  public void getApplicationShouldReturnNullWhenThereIsNoApplicationWithTheGivenId() {
+    //Given
+    given(applicationsRepository.findOne(APPLICATION_ID)).willReturn(applicationEntity);

```

Comment:
This statement contradicts with the test name.

If the test name represents what you want to achieve, then repositories findOne method should return null (as it would for a real repository).
Also there is a converter call in the code which is not mocked, that's why null was returned.